lib/util/region.ts

```diff
@@ -0,0 +1,12 @@
+export class Region {
```

Comment:
Does this need to be a class? Can it just be an interface?

Reply:
i was thinking that as well, but the connection failures property might be used in the future, i'm not sure if it's actually necessary, so right now i just have it set to 0, but it might be one day and interfaces can't have a property initialized already inside them, so i just made it a class

Reply:
Yeah sure, but you can assign the value when you instantiate the value, but fair enough :)

Reply:
yeah it's not a perfect solution cuz i don't really know what this'll look like until later, might change it, we'll see